docs/scos/dev/front-end-development/migration-guide-remove-support-of-ie11.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Migration guide - Remove support of IE11
+description: Use this guide to migrate the project from Yarn to NPM v8.
+template: concept-topic-template
+---
+
+This document provides instructions for removing support of IE11 in your Spryker project.
+
+## Overview
+
+Microsoft dropped the support of IE11 in June 2022. So it has to be removed in Spryker.
+
+*Estimated migration time: 1h*
+
+## Update modules
+
+Run the next command to update module versions:
+```bash
+composer update spryker/zed-ui spryker-shop/shop-ui
+```
+
+## Update configuration files
+
+`.browserslistrc`:
+
+Remove IE11 usage:
+```text
+IE 11
+```
+
+`frontend/configs/development.js`: 
+
+1. Remove import of `es6-polyfill` file: 
+```js
+'es6-polyfill': es6PolyfillTs,
+```
+
+2. Remove browsers support as it specified in the `.browserslistrc` file:
+```js
+browsers: [
+    '> 1%',
+    'ie >= 11',
+],
+```
+
+3. Remove `isES6Module` usage from conditions, like: 
+```js
+isES6Module ? `./js/${appSettings.name}.[name].js` : `./js/${appSettings.name}.[name].${buildVariant}.js`,
+    
+...
+
+...(!isES6Module ? ['@babel/plugin-transform-runtime'] : []),
```

Comment:
Also should be replaced by `['@babel/plugin-transform-runtime']` instead of removing